frontend/pages/shopping-lists/_id.vue

```diff
@@ -56,14 +108,25 @@
 
       <!-- View By Label -->
       <div v-else>
-        <div v-for="(value, key, idx) in itemsByLabel" :key="key" class="mb-6">
+        <div v-for="(value, key) in itemsByLabel" :key="key" class="mb-6">
           <div @click="toggleShowChecked()">
-            <span v-if="idx || key !== $tc('shopping-list.no-label')">
-              <v-icon :color="getLabelColor(value[0])">
-                {{ $globals.icons.tags }}
-              </v-icon>
-            </span>
-            {{ key }}
+            <!-- Create Item -->
```

Comment:
i think it is clear that this needs to be polished quite a bit, but i really like the concept and think that this creation flow is the way to go 👍